src/neptune_sacred/impl/__init__.py

```diff
@@ -114,7 +119,7 @@ def completed_event(self, stop_time, result: dict):
                     self._run[self.base_namespace][f"metrics/results/{k}"] = float(v)
                 elif isinstance(v, list) or isinstance(v, dict):
                     self._run[self.base_namespace][f"metrics/results/{k}"] = json.dumps(v)
-                elif isinstance(v, object):
+                elif isinstance(v, File):
```

Comment:
Would it make sense to add it to CHANGELOG that now we only log few selected types. (object would probably be catch-all, though I doubt that we logged anything meaningful.)